src/text.jl

```diff
@@ -0,0 +1,33 @@
+module Text
+
+    using ..lib
+    using ..utils
+
+    """Input command string for the DSS. (Getter)"""
+    function Command()::String
+        get_string(lib.Text_Get_Command())
+    end
+
+    """Input command string for the DSS. (Setter)"""
+    function Command(Value::String)::String
+        Value = readlines(IOBuffer(Value))
+        r = Vector{String}([])
+        for v in Value
+            lib.Text_Set_Command(v)
+            # TODO: lib.CheckForError()
+            push!(r, Result())
+        end
+        res = join(r, "\n") |> strip
+        if length(res) > 0:
```

Comment:
Pythonism!

Reply:
Haha oops, I've been outed. Fixed in b81533a.